osl/myolegs_osl.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" ?>
+<mujoco model="OSL_MyoLegs_v0.0.1">
```

Comment:
check if the model versioning is compatible through out the repo. I think there are different versioning methods in this repo. This might need harmonization

Reply:
In Myoleg, this is used `model="MyoLegs_v0.0.1"`. I am following this format